cmd/integration_test/build/buildspecs/test-eks-a-cli.yml

```diff
@@ -4,6 +4,7 @@ env:
   variables:
     INTEGRATION_TEST_MAX_EC2_COUNT: 45
     T_VSPHERE_CIDR: "198.18.0.0/16"
+    T_VSPHERE_PRIVATE_NETWORK_CIDR: "10.1.0.0/24"
```

Comment:
Do we need some extra VPN logic for this to work in our test environment?

Reply:
we will need to add vpn routes to the proper subnets. I will create a cr for it soon